app/folder/layout.tsx

```diff
@@ -0,0 +1,11 @@
+import BaseLayout from "@/components/layout/BaseLayout";
+import React from "react";
+
+type Props = {
+  children: React.ReactNode;
+};
+const FolderLayout = ({ children }: Props) => {
```

Comment:
타입이랑 컴포넌트 사이에 개행이 하나 들어가면 좋을 것 같아요~
```suggestion
};

const FolderLayout = ({ children }: Props) => {
```